kickoff/static/treestatus.js

```diff
@@ -0,0 +1,49 @@
+function _isTreeForThunderbird(treeName) {
+    return treeName.indexOf('comm') > -1;
+}
+
+function getTreeStatusUrl(branchName) {
+    var treeName = branchName.split('/')[1];
+    if (_isTreeForThunderbird(treeName)) {
+        treeName += '-thunderbird';
+    }
+
+    return 'https://treestatus.mozilla.org/' + treeName + '?format=json';
```

Comment:
------------------ Début de Rapport SpamAssassin ---------------------
Ce message est probablement du SPAM (message non sollicité envoyé en
masse, publicité, escroquerie...).

Cette notice a été ajoutée par le système d'analyse "SpamAssassin" sur
votre serveur de courrier "maralinga.yallingup.fr", pour vous
aider à identifier ce type de messages.

Le système SpamAssassin ajoute un en-tête "X-Spam-Flag: YES" aux
messages qu'il considère comme étant probablement du Spam.
Vous pouvez si vous le souhaitez utiliser cette caractéristique
pour régler un filtre dans votre logiciel de lecture de courrier,
afin de détruire ou de classer à part ce type de message.

Si ce robot a classifié incorrectement un message qui vous était
destiné, ou pour toute question, veuillez contacter l'administrateur
du système par e-mail à @@CONTACT_ADDRESS@@ .

Voir http://spamassassin.apache.org/tag/ pour plus de détails (en anglais).

Détails de l'analyse du message:   (6.3 points, 5.0 requis)
 0.8 BAYES_50               BODY: L'algorithme Bayésien a évalué la probabilité de spam
                            entre 40 et 60%
                            [score: 0.4920]
 1.3 RCVD_IN_BL_SPAMCOP_NET RBL: Relais listé dans
                            http://spamcop.net/bl.shtml
              [Blocked - see <http://www.spamcop.net/bl.shtml?195.154.220.89>]
 1.4 RCVD_IN_BRBL_LASTEXT   RBL: RCVD_IN_BRBL_LASTEXT
                            [195.154.220.89 listed in bb.barracudacentral.org]
 2.7 RCVD_IN_PSBL           RBL: Received via a relay in PSBL
                            [195.154.220.89 listed in psbl.surriel.com]

-------------------- Fin de Rapport SpamAssassin ---------------------